scripts/fonts.json

```diff
@@ -0,0 +1,450 @@
+{
+  "font-families": {
+    "Gothic A1": ["regular", "700"],
+    "M PLUS Rounded 1c": ["regular", "700"],
+    "Noto Sans": ["regular", "700", "italic", "700italic"],
+    "Noto Sans Arabic": ["regular", "700"],
+    "Noto Sans Armenian": ["regular", "700"],
+    "Noto Sans Bengali": ["regular", "700"],
+    "Noto Sans Devanagari": ["regular", "700"],
+    "Noto Sans Ethiopic": ["regular", "600"],
+    "Noto Sans Georgian": ["regular", "700"],
+    "Noto Sans Gurmukhi": ["regular", "700"],
+    "Noto Sans Hebrew": ["regular", "700"],
```

Comment:
Consider [Noto Rashi Hebrew](https://fonts.google.com/noto/specimen/Noto+Rashi+Hebrew) as an analogue to Noto Sans Italic.

Reply:
Noto Rashi Hebrew:

[![image](https://user-images.githubusercontent.com/3254090/225497779-c877f0dc-f6d7-4072-8928-e45565ceebde.png)](http://localhost:1776/#map=14/31.59067/34.54142&language=he)